xbmc/GUIInfoManager.cpp

```diff
@@ -2118,6 +2118,15 @@ const infomap videoplayer[] =    {{ "title",            VIDEOPLAYER_TITLE },
 /// @{
 /// \table_start
 ///   \table_h3{ Labels, Type, Description }
+///   \table_row3{   <b>`VideoFilter`</b>,
```

Comment:
Please disregard my previous comment. I didn't realise I had to compile Kodi with the changes introduced in this PR to test the infolabels (all working great now).

Please change their title in the docs for consistency when compared to all the infolabels/bools:
`VideoFilter` -> `RetroPlayer.VideoFilter`
`StretchMode` -> `RetroPlayer.StretchMode`
`VideoRotation` -> `RetroPlayer.VideoRotation`

In python, for example, we do `xbmc.getInfoLabel('retroplayer.stretchmode')` so this helps reducing confusion on how to use the label.